code/datums/achievements/viewer.dm

```diff
@@ -0,0 +1,47 @@
+//UI adapted from https://github.com/tgstation/tgstation/pull/47058
+
+/datum/achievement_browser
+	var/client/client
+	var/achievements = list()
```

Comment:
This variable isn't really being used to its full extent, probably worth making it a local variable just in the methods that use it.